docs/document/content/dev-manual/kernel.cn.md

```diff
@@ -32,13 +32,13 @@ chapter = true
 
 ## SQLExecutionHook
 
-| *SPI 名称*                     | *详细说明*              |
-| ----------------------------- | ---------------------- |
-| SQLExecutionHook              | SQL执行过程监听器         |
+| *SPI 名称*                     | *详细说明*      |
+| ----------------------------- |-------------|
+| SQLExecutionHook              | SQL 执行过程监听器 |
 
-| *已知实现类*                    | *详细说明*               |
-| ----------------------------- | ----------------------- |
-| TransactionalSQLExecutionHook | 基于事务的SQL执行过程监听器 |
+| *已知实现类*                    | *详细说明*            |
+| ----------------------------- |-------------------|
```

Comment:
The spaces between the symbols `|-` are consistent with other codes.

Reply:
@yx9o 
I didn't get you

Reply:
Space between `|` and `-`.

Reply:
![image](https://user-images.githubusercontent.com/12792261/158522512-185f98c6-dafe-47fb-ad1e-c81c93d53935.png)